AWESOME.md

```diff
@@ -50,7 +50,7 @@ because...
 <!--References-->
 [_brwf]: https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows
 [_crcl]: https://circleci.com
-[_dirl]: https://www.nist.gov/director/licensing
+[_dirl]: https://www.nist.gov/director/copyright-fair-use-and-licensing-statements-srd-data-and-software
```

Comment:
I would 100% approve, with the addition of #software:
```
[_dirl]: https://www.nist.gov/director/copyright-fair-use-and-licensing-statements-srd-data-and-software#software
```

Reply:
To make it more confusing, there is an internal link as well for instructions to publish software with a link to an inet page with the license text, https://inet.nist.gov/adlp/open-access-research/publishing-instructions

The text inside the blue box, is the only text needed for the NIST license.md file.